spec/spec_helper.rb

```diff
@@ -24,7 +27,7 @@
   default_facts.merge!(YAML.load(File.read(File.expand_path('../default_facts.yml', __FILE__)))) if File.exist?(File.expand_path('../default_facts.yml', __FILE__))
   default_facts.merge!(YAML.load(File.read(File.expand_path('../default_module_facts.yml', __FILE__)))) if File.exist?(File.expand_path('../default_module_facts.yml', __FILE__))
   c.default_facts = default_facts
-  c.mock_with :mocha
```

Comment:
It can be and was being set...  incorrectly.
https://github.com/voxpupuli/puppet-network/pull/202